### PR TITLE
[1LP][RFR] Update aws_iam auth/RBAC test, new access tree in 510z

### PR DESCRIPTION
**Pull request by mshriver**
Update access control tree for 510z

BZ blocks 59z testing.

{{ pytest: --long-running  -vv cfme/tests/integration/test_aws_iam_auth_and_roles.py }}

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 99e09b726e2b4f08c8942369114f6a70474330de:

`cfme/base/__init__.py`:
- :bangbang: Line 136:29: **E261** *at least two spaces before inline comment*
- :bangbang: Line 136:30: **E262** *inline comment should start with '# '*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 7544c2ffe2ec99de3e60b44f3c815edc8e5ff06d:

`cfme/base/__init__.py`:
- :bangbang: Line 136:29: **E261** *at least two spaces before inline comment*
- :bangbang: Line 136:30: **E262** *inline comment should start with '# '*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

File: cfme/base/__init__.py [hunk 1]
```diff
@@ -133,7 +133,7 @@ def get_master(self):
             if self.appliance.version == LATEST or self.appliance.is_pod:
                 name = 'EVM'
             else:
-                name = server.name
+                name = 'EVM' #server.name  # TODO remove monkey patch so psav doesn't yell
```
Comment (mshriver) [inline cfme/base/__init__.py]: this should not be merged and is a workaround for PRT

**Comment by dajoRH** [comment]
Lint report for commit e87aede4725263e07aaf4b1ee5b6f6db5c9ef0b0:

`cfme/base/__init__.py`:
- :bangbang: Line 136:29: **E261** *at least two spaces before inline comment*
- :bangbang: Line 136:30: **E262** *inline comment should start with '# '*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 47f4bd75510a5bc50ac1dca5dbee062e7c7a32d3:

`cfme/base/__init__.py`:
- :bangbang: Line 136:29: **E261** *at least two spaces before inline comment*
- :bangbang: Line 136:30: **E262** *inline comment should start with '# '*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 06e0359d5b18542a6d0fbb33de21b628edc58f40:

`cfme/base/__init__.py`:
- :bangbang: Line 136:29: **E261** *at least two spaces before inline comment*
- :bangbang: Line 136:30: **E262** *inline comment should start with '# '*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 43cdd74eec2b554374d6b79886a0cb6ea55ce1a8:

`cfme/base/__init__.py`:
- :bangbang: Line 136:29: **E261** *at least two spaces before inline comment*
- :bangbang: Line 136:30: **E262** *inline comment should start with '# '*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 77ae1463051bebfccb6c5bce40c4fc2c58d3693b:

`cfme/base/__init__.py`:
- :bangbang: Line 136:29: **E261** *at least two spaces before inline comment*
- :bangbang: Line 136:30: **E262** *inline comment should start with '# '*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

File: cfme/fixtures/authentication.py [hunk 2]
```diff
@@ -19,6 +19,7 @@ def setup_aws_auth_provider(appliance, amazon_auth_provider):
     original_config = appliance.server.authentication.auth_settings
     appliance.server.authentication.configure(auth_mode='amazon',
                                               auth_provider=amazon_auth_provider)
+    sleep(60)  # wait for MIQ to update, no trigger to look for, but if you try too soon it fails
```
Comment (izapolsk) [inline cfme/fixtures/authentication.py]: Q: is there something to check instead of sleep ?

File: cfme/roles.py [hunk 3]
```diff
@@ -381,7 +526,7 @@ def _remove_from_all(roles, r_page):
             'Clouds': ['Instances'],
             'Infrastructure': ['Datastores', 'Providers', 'Virtual Machines', 'Hosts', 'Clusters',
                                'PXE', 'Resource Pools'],
-            'Physical Infrastructure': ['Servers', 'Providers']},
+            'Physical Infrastructure': ['Providers', 'Servers', 'Topology']},
```
Comment (izapolsk) [inline cfme/roles.py]: did they change order ?

File: cfme/fixtures/authentication.py (hunk 2 above)
Comment (mshriver) [reply cfme/fixtures/authentication.py]: There isn't, but I think I can drop this to 30s

File: cfme/roles.py (hunk 3 above)
Comment (mshriver) [reply cfme/roles.py]: IIRC they didn't change order, but this order is consistent with the UI.

**Pull request merged**